apps/meteor/app/file-upload/server/methods/sendFileMessage.ts

```diff
@@ -156,7 +163,7 @@ export const sendFileMessage = async (
 		msgData,
 	}: {
 		roomId: string;
-		file: Partial<IUpload>;
+		file: Partial<IUpload>[] | Partial<IUpload>;
```

Comment:
You can revert this since the way to send multiple files in a message will be only the normal message APIs (to confirm them), This method should be deprecated in favor of the normal send message API

Reply:
done sir